frontend/src/container/GridGraphLayout/WidgetHeader/index.tsx

```diff
@@ -200,4 +214,12 @@ function WidgetHeader({
 	);
 }
 
+WidgetHeader.defaultProps = {
+	onDelete: undefined,
+	onClone: undefined,
+	allowDelete: undefined,
```

Comment:
two default value in above this is true and here it is undefined

Reply:
Will change it to true.